sssd_test_framework/roles/ad.py

```diff
@@ -2114,4 +2145,238 @@ def lockout(self, duration: int, attempts: int) -> ADPasswordPolicy:
         return self
 
 
+class ADDNSServer(BaseObject[ADHost, AD]):
+    def __init__(self, role: AD):
+        """
+        :param role: AD host object.
+        :type role: ADHost
+        """
+        super().__init__(role)
+
+        self.domain: str = role.domain
+        """Domain name."""
+
+        self.server: str = role.server
+        """Server name."""
+
+    def zone(self, name: str) -> ADDNSZone:
+        """
+        Get ADDNsServerZone object.
+
+        :param name: Zone name.
+        :type name: str
+        :return: ADDNSServerZone object.
+        :rtype: ADDNSZone
+        """
+        return ADDNSZone(self.role, name)
+
+    def get_forwarders(self) -> list[str] | None:
+        """
+        Get DNS global forwarders.
+
+        :return: DNS global forwarders.
+        :rtype: list[str]
+        """
+        forwarders = self.host.conn.run("Get-DnsServerForwarder").stdout_lines
+        if forwarders is not None:
+            parsed_forwarders = attrs_parse(forwarders, ["IPAddress"])
```

Comment:
I thought `attrs_parse` was for LDAP search results only?

Reply:
Anything multilined, delimited by a semicolon.